models/oauth2.go

```diff
@@ -132,6 +132,9 @@ func GetActiveOAuth2Providers() ([]string, map[string]OAuth2Provider, error) {
 
 // InitOAuth2 initialize the OAuth2 lib and register all active OAuth2 providers in the library
 func InitOAuth2() error {
+	if err := oauth2.InitSigningKey(); err != nil {
```

Comment:
Merge it into oauth2.Init is better?

Reply:
I did not add it in the existing method because `oauth2.Init` is used for the other direction (Gitea -> other service). So I thought both methods should be logical separated.